templates/UmbracoExtension/Client/src/entrypoint.ts

```diff
@@ -0,0 +1,12 @@
+import { UmbEntryPointOnInit } from '@umbraco-cms/backoffice/extension-api';
+
+// load up the manifests here
+export const onInit: UmbEntryPointOnInit = (_host, _extensionRegistry) => {
+
+  console.log('Hello from my extension');
+
+  //// We can register many manifests at once via code
+  //// as opposed to a long umbraco-package.json file
+  //extensionRegistry.registerMany([
```

Comment:
Since we have a comment here anyways, maybe we should link to the docs on this? As a new user running the template may not know where to start from this point.

So probably either 
https://docs.umbraco.com/umbraco-cms/customizing/extending-overview/extension-types/backoffice-entry-point
https://docs.umbraco.com/umbraco-cms/customizing/extending-overview/extension-types/bundle 
depending on the decision for my other comment on the type 🙂